.github/workflows/app-near.yml

```diff
@@ -0,0 +1,54 @@
+name: Next app tests
+
+on:
+  pull_request:
+    branches: [main]
```

Comment:
:orange_circle: No check on feature branches? Is it to save computation resources?

Reply:
Yes but only on PR, not in regular commits